src/vs/workbench/parts/files/electron-browser/views/explorerView.ts

```diff
@@ -435,6 +435,7 @@ export class ExplorerView extends TreeViewsViewletPanel implements IExplorerView
 		// Update resource context based on focused element
 		this.disposables.push(this.explorerViewer.onDidChangeFocus((e: { focus: ExplorerItem }) => {
 			const isSingleFolder = this.contextService.getWorkbenchState() === WorkbenchState.FOLDER;
+			// 54483: Check with Isi
```

Comment:
@isidorn This depends on single folder state and we are planning to support opening an URI with different scheme in single folder. May I know this will impact the code here?

Reply:
@sandy081 this code is already URI ready and is not file scheme specific. Everything is good.

Reply:
Great. Thanks 👍